docs/hub/_toctree.yml

```
@@ -25,10 +25,16 @@
   - local: model-cards
     title: Model Cards
     sections:
+    - local: model-card-annotated
+      title: Annotated Model Card
     - local: model-cards-co2
       title: Carbon Emissions
-    - local: models-gated
-      title: Gated models
+    - local: model-card-guidebook
+      title: Model Card Guidebook
+    - local: model-card-landscape-analysis
+      title: Landscape Analysis 
+  - local: models-gated
+    title: Gating models
```

```suggestion
    title: Gated models
```
let's keep this title as it's consistent with datasets